src/services/queries/Users.js

```diff
@@ -15,17 +16,21 @@ export const getAllUsersQuery = gql`
 `;
 
 export const getUserSessionQuery = gql`
+  ${userInfo}
   query GetUserSession($password: String = "", $user_code: String = "") {
     users(
       where: { password: { _eq: $password }, username: { _eq: $user_code } }
     ) {
-      username
-      type
-      password
-      is_active
-      id
-      created_at
-      avatar_url
+      ...userInfo
+    }
+  }
+`;
+
+export const getUserByIdQuery = gql`
```

Comment:
Not sure if there is a standard for naming queries? @e-azocar do you know?

Reply:
I found this https://docs.devland.is/technical-overview/api-design-guide/graphql-naming-conventions